change credit pricing and rename running storage to active storage (#2523)
temporalio/documentation: docs-src/cloud/pricing-storage.md

An execution of a particular Workflow could exist for a few seconds, a day, month, or even forever. Temporal collects the Event History during this time and dispatches work when necessary. In this context, a Workflow Execution has only two states, open (active) or closed.

Storage costs are measured in gigabyte-hours (GBh) and include charges for active Workflows, active storage, and the long-term, "retained" storage of Event Histories of closed Workflows. These are measured per Namespace.

_Active storage_ is a measure of the amount of storage used to store active Workflows.

> When the execution of a Workflow ends, Temporal Cloud stores Event History for a defined Retention Period, for historical use. This is _retained storage_. Typical uses include compliance, debugging, workload refresh, and business analytics. Both kinds of storage have fixed costs.

Reviewer: ```suggestion
When the execution of a Workflow ends, Temporal Cloud stores the Event History per the Namespace's Retention Period. This is _retained storage_. Typical uses include compliance, debugging, workload refresh, and business analytics. Both kinds of storage have fixed costs.
```
